frontend/static/styles.css

```diff
@@ -300,6 +305,10 @@ body .ddoc .contextLink {
   color: theme("colors.jsr-cyan.700");
 }
 
+body .ddoc .breadcrumbs {
+  @apply flex-wrap;
+}
+
```

Comment:
This is the only new CSS; the rest of the changes in this file are just auto-formatting fixes.